### ISSUE TITLE
Retrieving a data attribute or class when using multiple instances

### ISSUE THREAD

**Issue by dericcain** [Hacker0x01/react-datepicker#876]
I have a table that has multiple instances of the datepicker. I am adding a class to each `DatePicker` with a unique `id` (i.e., `class="datepicker-12"`) so that when a date is selected, I can fire off a request to update the date on that specific entity. 

The problem that I am running into is when using `onChange`, the `event.target` is the actual day element on the calendar that was clicked and not the input (where my class with the unique id resides) that holds the actual date. I could use `onBlur`, but then a user could select a date and then the next action could be them navigating from the page. That means that the request would fire off, but I want to let them know as soon as they have changes the date that it did, in fact, update.

So, basically, I need a way to attach some unique attribute so that when a date is updated, I know which entity needs to be updated.  I have read through the docs but I cannot seem to find something like this and surely others have had to do this.

**Comment by nicolaskern** [comment]
I'm actually facing the same problem!

**Comment by nimesha95** [comment]
Any updates on this issue?
